timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/bdbdatafetchers/datafetchers/PaginationHelper.java

```diff
@@ -36,8 +42,8 @@
       );
     } else {
       return PaginatedList.create(
-        cursors[0],
-        cursors[1],
+        startedFromCursor ? cursors[0] : null,
```

Comment:
Cursors will benefit from a more descriptive name like "prevCursor" and "nextCursor"